holoviews/tests/plotting/plotly/testplot.py

```diff
@@ -123,3 +193,126 @@ def test_layout_instantiate_subplots_transposed(self):
         plot = plotly_renderer.get_plot(layout(plot=dict(transpose=True)))
         positions = [(0, 0), (0, 1), (1, 0), (1, 1), (2, 0), (2, 1), (3, 0), (3, 1)]
         self.assertEqual(sorted(plot.subplots.keys()), positions)
+
+
+@attr(optional=1)
```

Comment:
We have plans to replace these since they're kind of useless (in fact we plan to migrate to pytest at some point). For now I'd recommend following a pattern where you create a baseclass like this for all plotly tests (you can look in ``holoviews/tests/plotting/bokeh/testplot.py`` to see an example):

```python
class TestPlotlyPlot(ComparisonTestCase):

    def setUp(self):
        self.previous_backend = Store.current_backend
        self.comm_manager = bokeh_renderer.comm_manager
        bokeh_renderer.comm_manager = comms.CommManager
        if not bokeh_renderer:
            raise SkipTest("Plotly required to test plot instantiation")
        Store.current_backend = 'plotly'

    def tearDown(self):
        Store.current_backend = self.previous_backend
        bokeh_renderer.comm_manager = self.comm_manager
        Callback._callbacks = {}
```

Reply:
It looked to me like the `TestPlotlyPlotInstantiation` class above follows this pattern, and this is where I put the new tests that use the plotly backend.  I didn't follow the pattern for `TestPlotlyFigureGrid` because it's not actually testing the backend directly, just the new `figure_grid` util function.  Does that make sense?

Would it make sense to just remove the `@attr` for `TestPlotlyFigureGrid`?

Reply:
Ah, yes, totally missed that thanks. I think it's fine, I'll replace all the @attr decorators in one go anyway.